### PR TITLE
Examples module header

### PR DESCRIPTION
**Pull request by ChristianSonderUniSG**
# Pull request

## Types of changes

- [x] New feature (non-breaking change which adds functionality).
- [ ] Enhancement (non-breaking change which enhances functionality)
- [ ] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [ ] I have read the **[README](./README.md)** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/schema/examples/availability.xml
Comment (Bpolitycki) [inline src/schema/examples/availability.xml:1]: The example is not used anywhere and the example `xml:id` should not be direct party of the example snippet.

Comment (Bpolitycki) [inline src/schema/examples/availability.xml:1]: The example id should not be part of the example snippet itself.

File: src/schema/examples/seriesStmt.xml
Comment (Bpolitycki) [inline src/schema/examples/seriesStmt.xml:1]: May be valid ... but it would be better to use real UUID here.

**Review by Bpolitycki — CHANGES_REQUESTED** [review]
Use a real UUID in `<seriesStmt/>`...

File: src/schema/examples/seriesStmt.xml
Comment (ChristianSonderUniSG) [reply src/schema/examples/seriesStmt.xml:1]: Done

**Review by Bpolitycki — APPROVED** [review]

**Pull request merged**